commit e62a641de17b172ffc4d3a803085c8afbfbec3d1
Author: Florian La Roche <laroche@redhat.com>
Date:   Sun Feb 3 12:38:46 2008 +0100

    gitweb: Make feed entries point to commitdiff view
    
    Change feeds entries (feeds items) from pointing (linking) to 'commit'
    view to pointing to 'commitdiff' view.
    
    First, feed entries have whatchanged-like list of files which were
    modified in a commit, so 'commitdiff' view more naturally reflects
    feed entry (is more naturally alternate / extended version of a feed
    item). Second, this way the patches are shown directly and code review
    is done more easily via watching feeds.
    
    [jn: Rewritten commit message]
    
    Signed-off-by: Florian La Roche <laroche@redhat.com>
    Signed-off-by: Jakub Narebski <jnareb@gmail.com>
    Signed-off-by: Junio C Hamano <gitster@pobox.com>

diff --git a/gitweb/gitweb.perl b/gitweb/gitweb.perl
index ae2d057..8ef2735 100755
--- a/gitweb/gitweb.perl
+++ b/gitweb/gitweb.perl
@@ -5565,7 +5565,7 @@ XML
 			or next;
 
 		# print element (entry, item)
-		my $co_url = href(-full=>1, action=>"commit", hash=>$commit);
+		my $co_url = href(-full=>1, action=>"commitdiff", hash=>$commit);
 		if ($format eq 'rss') {
 			print "<item>\n" .
 			      "<title>" . esc_html($co{'title'}) . "</title>\n" .
